jaxdecomp/_src/padding.py

```diff
@@ -33,7 +43,25 @@ def impl(arr: ArrayLike,
            padding_width: int | tuple[int],
            pdims: tuple[int],
            mode: str = 'constant') -> Array:
-
+    """
+        Implementation of the slice padding operation.
+
+        Parameters
+        ----------
+        arr : ArrayLike
+            Input array to be padded.
+        padding_width : int | tuple[int]
+            Width of padding to apply.
+        pdims : tuple[int]
```

Comment:
Why do we need to know pdims here?

Reply:
I am gonna delete these two
I have implemented a much better padding unpadding and halo reduce in Jax PM

Reply:
I will keep them for now (because they were included in the JOSS paper)
But this is the unpadding and reduce halo that should be used 
https://github.com/DifferentiableUniverseInitiative/JaxPM/blob/distributed_pm/jaxpm/_src/base_ops.py#L78